### ISSUE TITLE
Remote Debugging with Rider IDE

### ISSUE THREAD

**Issue by DerekMelchin** [QuantConnect/Documentation#958]
#### Expected Behavior

The [CLI docs](https://www.quantconnect.com/docs/v2/lean-cli/backtesting/debugging#05-C-and-Rider) explain how to set up debugging to the remote host with Rider.
> It would be nice to have links or a description in the section about setting up rider for debugging, with regards to the remote host `root@localhost:2222` I have added one but can't see any help on what the ssh settings should be or credentials that should be used to allow it to connect to the docker container that lean-cli created, I will keep searching your docs, maybe I missed it.

--

> thanks for the reply, it's ok I figured it out, the ssh private key was in the lean-cli logs, I just didn't see it as I had a small window at the time, I guess the docs on you site could mention that the ssh key will be in the logs incase anyone else misses it to.

#### Actual Behavior

The docs don't include this information.
Waiting on https://github.com/QuantConnect/lean-cli/issues/171

#### Checklist

- [x] I have completely filled out this template
- [x] I have confirmed that this issue exists on the current `master` branch
- [x] I have confirmed that this is not a duplicate issue by searching [issues](https://github.com/QuantConnect/Lean/issues)
